Jenkinsfile

```diff
@@ -138,7 +139,9 @@ pipeline {
             steps {
                 milestone 1
                 container('node') {
-                    sh "yarn lerna publish --registry https://registry.npmjs.org/"
+                    sh "set +x; npm set //registry.npmjs.org/:_authToken ${NPM_TOKEN}"
```

Comment:
semantic release library does this in https://github.com/semantic-release/npm/blob/1f3ab84a5d05947960be9a7c78642e13fb516496/lib/set-npmrc-auth.js but lerna apperantly uses something else.